lib/AS_BH1750/AS_BH1750A.cpp

```diff
@@ -359,12 +359,12 @@ uint16_t AS_BH1750A::readRawLevel(void) {
   level <<= 8;
   level |= Wire.receive();
 #endif
-  if(Wire.endTransmission()!=0) {
-#if BH1750_DEBUG == 1
-    Serial.println("I2C read error");
-#endif
-    return 65535; // Error marker
-  }
+//   if(Wire.endTransmission()!=0) {  // Issue #1417
+// #if BH1750_DEBUG == 1
+//     Serial.println("I2C read error");
```

Comment:
Hmm this is also quite a serious one, as a library should never write to Serial.
We do have several plugins that may interact with the serial port which means this library may put a connected sensor into an unknown state.

Reply:
Well, in some places that serial debug even is rather misleading... but it helped me verify it is working.